drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -230,6 +232,11 @@ struct lis2dh_data {
 	struct sensor_value temperature;
 #endif
 
+#ifdef CONFIG_PM_DEVICE
+	enum pm_device_state power_state;
```

Comment:
please don't store local power states.

Reply:
Meant to remove remove that already, thanks.